src/qt/guiutil.cpp

```diff
@@ -766,10 +766,10 @@ QString NetworkToQString(Network net)
     assert(false);
 }
 
-QString ConnectionTypeToQString(ConnectionType conn_type, bool relay_txes)
+QString ConnectionTypeToQString(ConnectionType conn_type, bool relay_txes, bool relevant_services)
 {
     switch (conn_type) {
-    case ConnectionType::INBOUND: return relay_txes ? QObject::tr("Inbound Full Relay") : QObject::tr("Inbound Block Relay");
+    case ConnectionType::INBOUND: return !relay_txes && relevant_services ? QObject::tr("Inbound Block Relay") : QObject::tr("Inbound Full Relay");
```

Comment:
spv nodes aren't full relay peers either if they have tx relay disabled

Reply:
thanks for having a look...simplified to "Inbound" and "Inbound Block Relay", open to other suggestions you have